libcxx/test/std/algorithms/alg.nonmodifying/alg.find.last/ranges.find_last_if.pass.cpp

```diff
@@ -0,0 +1,263 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Would be good to have a `test_deque()`-equivalent in this file.

Reply:
I actually don't think it's necessary - I think it's just a leftover from when `ranges::find` was broken with `deque`. No reason to spend the compute cycles.